libs/block-content/website/src/lib/teaser/teaser.tsx

```diff
@@ -185,7 +217,68 @@ export const TeaserAuthors = styled('div')`
 
 export const TeaserContent = styled(TeaserInnerWrapper)``
 
+export const ImagePlaceholder = styled('div')`
+  width: 100%;
+  object-fit: cover;
+  grid-column: 1/13;
+  aspect-ratio: 1/1;
+`
+
+export const TeaserContentStyled = styled(TeaserContent)`
+  grid-column: 1/13;
+`
+
+export const TeaserPreTitleNoContent = styled(TeaserPreTitle)<{isHovered: boolean}>`
+  transition: background-color 0.3s ease-in-out;
+  background-color: ${({theme, isHovered}) =>
+    isHovered ? theme.palette.primary.main : theme.palette.common.black};
+  height: 3px;
+  width: 100%;
+  margin-bottom: ${({theme}) => theme.spacing(1.5)};
+`
+
+export const TeaserPreTitleStyled = styled(TeaserPreTitle)<{isHovered: boolean}>`
+  transition: background-color 0.3s ease-in-out;
+  background-color: ${({theme, isHovered}) =>
+    isHovered ? theme.palette.primary.main : theme.palette.secondary.main};
```

Comment:
dont think thats part of the design with the hover.

Reply:
![Uploading Screenshot 2024-04-03 at 11.00.55.png…]()